src/contracts/security/security.service.spec.ts

```diff
@@ -344,7 +275,8 @@ describe('SecurityService', () => {
 
       expect(mockPauseDeposits).toBeCalledTimes(1);
       expect(mockWait).toBeCalledTimes(1);
-      expect(mockGetPauseMessagePrefix).toBeCalledTimes(1);
+      // 1 - repository, 2 — signPauseData, 3 — pauseDeposits
```

Comment:
The same, please provide more comments about calls

Reply:
fixed